gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-ui.yaml

```diff
@@ -127,6 +127,59 @@ paths:
             $ref: "#/components/responses/ThemesResponse"
           default:
             $ref: "#/components/responses/Error"
+
+    /environments/{envId}/ui/themes/_default:
+      servers:
+        - url: "{protocol}://{managementAPIHost}/management/v2/organizations/{orgId}"
+          description: APIM Management API v2 - Base URL to target specific organizations
+          variables:
+            protocol:
+              description: The protocol you want to use to communicate with the mAPI
+              default: https
+              enum:
+                - https
+                - http
+            managementAPIHost:
+              description: The domain of the server hosting your Management API
+              default: localhost:8083
+            orgId:
+              description: The unique ID of your organization
+              default: DEFAULT
+        - url: "{protocol}://{managementAPIHost}/management/v2"
```

Comment:
#question
Why add this part? Can't it be done once as a whole?

Reply:
It is included so that the user can either call : `environments/:envId/themes/_default` or `organizations/:orgId/environments/:envId/themes/_default`.